### PR TITLE
fix(addon-table): `table` controls do not work in heading

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Part of #5436

### CODE REVIEW

File: projects/addon-table/components/table/th/th.style.less [hunk 1]
```diff
@@ -15,6 +15,8 @@
     box-sizing: border-box;
     box-shadow: 0 0.3125rem rgba(237, 237, 237, 0);
     border: 1px solid var(--tui-base-04);
+    // Create new viewport for fixed positioning
+    transform: translate3d(0, 0, 0);
```
Comment (waterplea) [inline projects/addon-table/components/table/th/th.style.less:19]: There's also padding reset in TD if there's a control inside.

Comment (splincode) [reply projects/addon-table/components/table/th/th.style.less:19]: fixed

Comment (waterplea) [reply projects/addon-table/components/table/th/th.style.less:19]: Where does the editable class come from?

Comment (splincode) [reply projects/addon-table/components/table/th/th.style.less:19]: fixed

**Review by waterplea — APPROVED** [review]

**Review by waterplea — COMMENTED** [review]
Come to think of it, if somebody has some unrelated NgControl in their headers it would now break their padding. We probably should remove the padding part and control query.

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**